x-pack/plugins/upgrade_assistant/public/components/tabs/overview/steps.tsx

```diff
@@ -0,0 +1,196 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { Fragment, StatelessComponent } from 'react';
+
+import {
+  EuiFormRow,
+  EuiFormRowProps,
+  EuiLink,
+  EuiNotificationBadge,
+  EuiSpacer,
+  // @ts-ignore
+  EuiStat,
+  EuiSteps,
+  EuiText,
+} from '@elastic/eui';
+import { FormattedMessage, injectI18n } from '@kbn/i18n/react';
+
+import { NEXT_MAJOR_VERSION } from '../../../version';
+import { UpgradeAssistantTabProps } from '../../types';
+import { DeprecationLoggingToggle } from './deprecation_logging_toggle';
+
+// Eui's types don't have this prop
+const EuiFormRowPrime: React.StatelessComponent<
+  EuiFormRowProps & { describedByIds?: string[] }
+> = EuiFormRow;
+
+export const StepsUI: StatelessComponent<
+  UpgradeAssistantTabProps & ReactIntl.InjectedIntlProps
+> = ({ checkupData, setSelectedTabIndex, intl }) => {
+  const countByType = Object.keys(checkupData!).reduce(
+    (counts, checkupType) => {
+      counts[checkupType] = checkupData![checkupType].length;
+      return counts;
+    },
+    {} as { [checkupType: string]: number }
+  );
+
+  return (
+    <EuiSteps
+      className="upgSteps"
+      headingElement="h2"
+      steps={[
+        {
+          title: intl.formatMessage({
+            id: 'xpack.upgradeAssistant.overviewTab.steps.clusterStep.stepTitle',
+            defaultMessage: 'Check for issues with your cluster',
```

Comment:
Can we make this messaging more succinct? Something like:

<img width="493" alt="screen shot 2018-12-11 at 10 30 19 am" src="https://user-images.githubusercontent.com/549577/49811004-55877b80-fd30-11e8-8a34-8c44f868fe52.png">

^^ @gchaps ?